crates/symbolicator/src/metrics.rs

```diff
@@ -1,16 +1,20 @@
 //! Provides access to the metrics sytem.
 use std::net::ToSocketAddrs;
+use std::ops::{Deref, DerefMut};
 use std::sync::Arc;
 
-use cadence::{StatsdClient, UdpMetricSink};
+use cadence::{Metric, MetricBuilder, StatsdClient, UdpMetricSink};
 use parking_lot::RwLock;
 
+type Hostname = String;
+type HostnameTag = String;
```

Comment:
i'm curious why you chose aliases rather than newtypes?

Reply:
I just wanted it to be clear which string is which in the definition of `MetricsClient`. I don't think a newtype would provide any extra safety in this situation.